runtime/compiler/env/j9method.h

```diff
@@ -518,17 +525,21 @@ class TR_ResolvedJ9Method : public TR_J9Method, public TR_ResolvedJ9MethodBase
    virtual void makeParameterList(TR::ResolvedMethodSymbol *methodSym);
 
    /**
-    * @brief Check if a field is a QType or not.
+    * @brief Check if a field is with NullRestricted attribute.
```

Comment:
"if a field is with" would read better as "if a field has the" or maybe "if this is a field with the"

Reply:
Addressed in 32f6437